### ISSUE TITLE
What to use instead of InjectedProps

### ISSUE THREAD

**Issue by abeMedia** [icd2k3/react-router-breadcrumbs-hoc#113]
The last version saw the TS definition for `InjectedProps` get removed. What are we meant to use instead?

**Comment by asherccohen** [comment]
Any idea how this can be fixed?
Breadcrumbs assume a React.ReactNode[] but with no further type definition of breadcrumbs.

Any help?

**Comment by icd2k3** [comment]
Sorry, this slipped through the cracks - happy to accept a PR for updated types, or I may have some time soon to update this.

**Comment by icd2k3** [comment]
Hello @abeMedia @asherccohen  👋 . I switched from manually defining types to having them generated for the package (when I switched the src code from js to ts, this is probably when we switched off of `InjectedProps`). The current definition for a breadcrumbs route is:

```js
export interface BreadcrumbsRoute {
    path: string;
    breadcrumb?: React.ComponentType | React.ElementType | string;
    matchOptions?: MatchOptions;
    routes?: BreadcrumbsRoute[];
}
```

Where breadcrumb is an _optional_ component/element/string. 

Is this missing something? Let me know and I'm happy to adjust or accept a PR!
